src/graphic/Fast3D/gfx_sdl2.cpp

```diff
@@ -358,7 +358,11 @@ static void gfx_sdl_init(const char* game_name, const char* gfx_api_name, bool s
     height = window_height;
 #endif
 
+#ifdef PLATFORM_IOS
+    Uint32 flags = SDL_WINDOW_BORDERLESS | SDL_WINDOW_SHOWN;
+#else
     Uint32 flags = SDL_WINDOW_SHOWN | SDL_WINDOW_RESIZABLE | SDL_WINDOW_ALLOW_HIGHDPI;
+#endif
```

Comment:
any idea why ios needs this but android doesn't?

Reply:
SDL_WINDOW_ALLOW_HIGHDPI is broken on ios rn